examples/master.linen.conflist

```diff
@@ -0,0 +1,42 @@
+{
+   "name":"linen-network",
+   "cniVersion": "0.3.1",
+   "plugins":[
+      {
+         "type":"bridge",
+         "bridge":"kbr0",
```

Comment:
In this example `kbr0` is not the default bridge name. If we don't set bridge name here then we'll get `cni0` by default.